### PR TITLE
Added support for iPhone5,4(A1507) iOS 9.3.2 (Frisco 13F69)

### PR DESCRIPTION
**Pull request by icrackthecode**
Added support for iPhone5,4(A1507) iOS 9.3.2 (Frisco 13F69)

### CODE REVIEW

File: Trident.xcodeproj/project.xcworkspace/contents.xcworkspacedata [hunk 1]
```diff
@@ -2,6 +2,6 @@
 <Workspace
```
Comment (benjamin-42) [inline Trident.xcodeproj/project.xcworkspace/contents.xcworkspacedata:2]: Please do not commit this file

File: Trident/offsetfinder.c [hunk 2]
```diff
@@ -16,6 +16,7 @@
 t_target_environment target_environment = NotSupported;
 
 t_target_environment info_to_target_environment(const char *device_model, const char *system_version) {
+    determineTarget("iPhone5,4", "9.3.2", iPhone52_iOS921);
```
Comment (benjamin-42) [inline Trident/offsetfinder.c]: Can you insert your device after iPhone53_iOS933 everywhere please? We try to keep offsetfinder ordered.

Comment (benjamin-42) [reply Trident/offsetfinder.c]: Also check the indentation, should be spaces

**Comment by benjamin-42** [comment]
Still conflicts also

File: Trident/offsetfinder.c (hunk 2 above)
Comment (icrackthecode) [reply Trident/offsetfinder.c]: okay sure let me work on that

**Pull request closed without merging**